src/style.css

```
@@ -0,0 +1,49 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
```

```suggestion
.stars__star {
  background-repeat: no-repeat;
  background-position: center;
  background-image: url(./images/star.svg);
  width: 16px;
  height: 16px;
  margin-right: 4px;
}
```
lets move default non active star to common selector